### PR TITLE
feat(plugins): render-markdown.nvim support

### PR DESCRIPTION
**Pull request by A-Lamia**
adds support for render-markdown.nvim this is an initial support may stay as is or some changes may be made later depending on how i feel with home things look.

### CODE REVIEW

**Review by mehalter — COMMENTED** [review]
Should a lot of these just link to the treesitter highlight groups?

**Comment by A-Lamia** [comment]
Your right, it's fixed

File: lua/astrotheme/groups/plugins/render-markdown.lua [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      bold = true,
+    },
+
+    RenderMarkdownCode = { bg = c.ui.tool },
+    RenderMarkdownCodeInline = { fg = c.syntax.red },
+    RenderMarkDownBullet = { fg = c.ui.orange, bold = true },
+    RenderMarkdownChecked = { fg = c.ui.cyan, bold = true },
+    RenderMarkdownUnchecked = { fg = c.ui.blue, bold = true },
+    RenderMarkdownTodo = { fg = c.ui.green, bold = true },
+    RenderMarkDownLink = { fg = c.ui.blue, bold = true },
```
Comment (mehalter) [inline lua/astrotheme/groups/plugins/render-markdown.lua:50]: should these as well @A-Lamia ?

Comment (A-Lamia) [reply lua/astrotheme/groups/plugins/render-markdown.lua:50]: I did those and i checked and i did change them but i added the whole file so I'm insane or something bugged lol.

**Review by mehalter — APPROVED** [review]

**Pull request merged**